frontend/app/components/wp-edit/wp-edit-field/wp-edit-field.directive.ts

```diff
@@ -218,10 +225,17 @@ function wpEditFieldLink(scope,
                          attrs,
                          controllers: [WorkPackageEditFormController, WorkPackageEditFieldController]) {
 
-  controllers[1].formCtrl = controllers[0];
-  controllers[1].formCtrl.registerField(scope.vm);
+  var formCtrl = controllers[0];
+  controllers[1].formCtrl = formCtrl;
+
+  formCtrl.registerField(scope.vm);
+  formCtrl.onWorkPackageUpdated('wp-edit-field-' + scope.vm.fieldName, (wp) => {
+    scope.vm.initializeField(wp);
```

Comment:
I think we should switch to a getter/setter for this.work_package and set the value here before calling `initializeField` (then not needing to pass the `wp` any more). With the change at hand we now have two ways in which a work package is passed to the directive:
1) via the watcher defined here
2) via the `workPackage` getter accessing the form's workPackage

We can assume the two methods to deliver the same results but that is only an assumption.